tests/display/test_inputs.py

```diff
@@ -80,6 +95,7 @@ class TestListInput:
         (["abc", "def", "ghi"], "def") # default not first value
     ])
     def test_init_from_param(self, qtbot, choices, default_value):
+        # First test is with value = default value
```

Comment:
as above, adding a `parametrize` like ```@pytest.mark.parametrize("reference", ['default', 'second_choice'])``` and adapting the logic should do the trick.